api/apiclient.go

```diff
@@ -806,6 +806,7 @@ type dialer struct {
 // when appropriate.
 func (d dialer) dial(_ <-chan struct{}) (io.Closer, error) {
 	a := retry.StartWithCancel(d.openAttempt, d.opts.Clock, d.ctx.Done())
+	var lastErr error = nil
```

Comment:
Out of curiosity what's the reason for setting `= nil`, as `var lastErr error` will be nil anyway?

Reply:
Not particularly needed. I think this started out as some sort of "lastErr := nil", but since that isn't a typed nil, I ended up with var ... error, and left the '='.